docs/nodes/NODES.md

```diff
@@ -0,0 +1,68 @@
+
+## Important Concepts
+
+There are several node grouping concepts that can be examined with a narrow focus. These (and other) groupings can be pieced together to make up functional graph setups, and are important to understanding how groups of nodes work together as part of a whole. Note that the screenshots below aren't examples of complete functioning node graphs (see Examples).
+
+### Noise
+
+An initial noise tensor is necessary for the latent diffusion process. As a result, the Denoising node requires a noise node input.  
+
+![groupsnoise](../assets/nodes/groupsnoise.png)
+
+### Conditioning
+
+Conditioning is necessary for the latent diffusion process, whether empty or not. As a result, the Denoising node requires positive and negative conditioning inputs. Conditioning is reliant on a CLIP text encoder provided by the Model Loader node.
+
+![groupsconditioning](../assets/nodes/groupsconditioning.png)
+
+### Image Space & VAE
+
+The ImageToLatents node doesn't require a noise node input, but requires a VAE input to convert the image from image space into latent space. In reverse, the LatentsToImage node requires a VAE input to convert from latent space back into image space.
+
+![groupsimgvae](../assets/nodes/groupsimgvae.png)
+
+### Defined & Random Seeds
+
+It is common to want to use both the same seed (for continuity) and random seeds (for variance). To define a seed, simply enter it into the 'Seed' field on a noise node. Conversely, the RandomInt node generates a random integer between 'Low' and 'High', and can be used as input to the 'Seed' edge point on a noise node to randomize your seed.
+
+![groupsrandseed](../assets/nodes/groupsrandseed.png)
+
+### Control
+
+Control means to guide the diffusion process to adhere to a defined input or structure. Control can be provided as input to non-image *ToLatents nodes from ControlNet nodes. ControlNet nodes usually require an image processor which converts an input image for use with ControlNet.
+
+![groupscontrol](../assets/nodes/groupscontrol.png)
+
+### LoRA
+
+The Lora Loader node lets you load a LoRA (say that ten times fast) and pass it as output to both the Prompt (Compel) and non-image *ToLatents nodes. A model's CLIP tokenizer is passed through the LoRA into Prompt (Compel), where it affects conditioning. A model's U-Net is also passed through the LoRA into a non-image *ToLatents node, where it affects noise prediction.
+
+![groupslora](../assets/nodes/groupslora.png)
+
+### Scaling
+
+Use the ImageScale, ScaleLatents, and Upscale nodes to upscale images and/or latent images. The chosen method differs across contexts. However, be aware that latents are already noisy and compressed at their original resolution; scaling an image could produce more detailed results.
```

Comment:
I’m not sure what the take-home message is here. Is it a good or a bad thing that scaling the image can produce more detailed results?

Also the term “upscaling” may be unfamiliar to readers. Could you describe this as “enlarging the image and adding more detail?”

Reply:
Updated!